jwk/jwt_strategy.go

```diff
@@ -114,12 +116,12 @@ func (j *RS256JWTStrategy) refresh(ctx context.Context) error {
 		return err
 	}
 
-	public, err := FindKeyByPrefix(keys, "public")
+	public, err := FindPublicKey(keys)
```

Comment:
I really love that you removed this public private key prefix stuff. Thank you!

Reply:
Thanks! :) It really helped to get rid of some logic errors because key pairs on hsm are always with matching id's so they cannot contain prefixes.